README.md

```diff
@@ -5,7 +5,7 @@ An Omeka-S module which adds the required Javascript for Google Analytics tracki
 
 Currently supported versions of Omeka-S: 1.0.0beta1 - 1.0.0beta5
 
-[![Build Status](https://travis-ci.com/digirati-co-uk/omeka-google-analytics-module.svg?token=WyNXjcfGtVVkNhxsZzyy&branch=master)](https://travis-ci.com/digirati-co-uk/omeka-google-analytics-module)
```

Comment:
good catch, I dont think that token allows anything else other than the badge lookups will fix tho